app/src/components/pages/ModifyProfile.jsx

```diff
@@ -0,0 +1,197 @@
+'use client';
+import React, { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import { getAuth, onAuthStateChanged } from "firebase/auth";
+import { getFirestore, doc, getDoc, updateDoc } from "firebase/firestore";
+import { Button } from "@/components/ui/button";
+import { Card } from "@/components/ui/card";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
+import { Select, SelectContent, SelectItem, SelectTrigger, SelectValue } from "@/components/ui/select";
+import { Textarea } from "@/components/ui/textarea"; // Importa il componente Textarea
+import Header from "@/components/ui/header";
+// Importa i moduli necessari da Firebase v9+
+import { getStorage, ref, uploadBytes, getDownloadURL } from 'firebase/storage';
+
+
+export default function MentorProfileForm() {
+  const [user, setUser] = useState(null);
+  const [userData, setUserData] = useState(null);
+  const [formState, setFormState] = useState({});
+  const [loading, setLoading] = useState(true);
+  const [cvFile, setCvFile] = useState(null); // Stato per il file del CV
+  const [portfolioFiles, setPortfolioFiles] = useState([]); // Stato per i file del portfolio
+  const navigate = useNavigate();
+  const auth = getAuth();
+  const db = getFirestore();
+  const storageRef = storage().ref();
```

Comment:
`storage()` non è definita, non è possibile usarla, credo che tu volessi usare `getStorage` in questo caso